test-helpers/index.js

```diff
@@ -47,9 +47,12 @@ const testHelpers = (module.exports = builders(schema, {
 }));
```

Comment:
I'd avoid adding unnecessary test helpers, you can just pass a paragraph with some random text and check if resulting document matches that